lib/redmine/menu_manager/menu_helper.rb

```diff
@@ -322,7 +322,7 @@ def allowed_node?(node, user, project)
   end
 
   def allowed_project_node?(node, project, user)
-    if node_action_allowed?(node, project, user)
+    if node_action_allowed?(node, project, user) || node.external_link?
```

Comment:
why not putting the check `node.external_link?` into `node_action_allowed?` ? Seems like the better place for it.